updater/scripts/api-requests.sh

```diff
@@ -5,7 +5,7 @@
 echo -e "resource\ttype\tsource IP\trequests/day"
 
 zcat -f /var/log/haproxy.log.1* |
-    perl -ne 'print if s/.*: ([^:]+).*\/api\/v3\/([^\/\? ]+)\/([^\/\? ]+?(\/[^\/\? ]+)).*/\1 \2 \3/' |
+    perl -ne 'print if s/.*?: ([^:]+).*\/api\/v3\/([^\/\? ]+)\/([^\/\? ]+?(\/[^\/\? ]+)).*/\1 \2 \3/' |
```

Comment:
This works, thank you 👍 ! However, I wonder if the following might be easier to understand (untested):
```
perl -ne 'print if s/.*haproxy\[\d+\]: ([^:]+).*\/api\/v3\/([^\/\? ]+)\/([^\/\? ]+?(\/[^\/\? ]+)).*/\1 \2 \3/' 
```